### ISSUE TITLE
BLC dependency not working on latest node

### ISSUE THREAD

**Issue by dbogatov** [stevenvachon/broken-link-checker#89]
Better see the terminal output.
That happens for every URL on node 9+ for BLC v0.7.6

```
$ docker run -it node:9.2.0-alpine /bin/sh
Unable to find image 'node:9.2.0-alpine' locally
9.2.0-alpine: Pulling from library/node
b56ae66c2937: Already exists
327e64e394fc: Pull complete
a52c21ec8ada: Pull complete
Digest: sha256:b5f561c53495f4dd66144645d36089ab50a30accae51ecf8b7920178a517ae48
Status: Downloaded newer image for node:9.2.0-alpine
/ # npm install -g broken-link-checker
npm WARN deprecated nopter@0.3.0: try optionator
/usr/local/bin/blc -> /usr/local/lib/node_modules/broken-link-checker/bin/blc
/usr/local/bin/broken-link-checker -> /usr/local/lib/node_modules/broken-link-checker/bin/blc
+ broken-link-checker@0.7.6
added 100 packages in 2.822s
/ # blc https://google.com
/usr/local/lib/node_modules/broken-link-checker/node_modules/urlcache/node_modules/urlobj/lib/parseUrl.js:124
			if (url.search[0] === "?")
			              ^

TypeError: Cannot read property '0' of null
    at Object.parseUrl [as parse] (/usr/local/lib/node_modules/broken-link-checker/node_modules/urlcache/node_modules/urlobj/lib/parseUrl.js:124:18)
    at parseUrl (/usr/local/lib/node_modules/broken-link-checker/node_modules/urlcache/index.js:107:16)
    at formatUrl (/usr/local/lib/node_modules/broken-link-checker/node_modules/urlcache/index.js:90:8)
    at UrlCache.set (/usr/local/lib/node_modules/broken-link-checker/node_modules/urlcache/index.js:72:8)
    at streamHtml (/usr/local/lib/node_modules/broken-link-checker/lib/internal/streamHtml.js:80:9)
    at item (/usr/local/lib/node_modules/broken-link-checker/lib/public/HtmlUrlChecker.js:34:4)
    at callHandler (/usr/local/lib/node_modules/broken-link-checker/node_modules/limited-request-queue/lib/index.js:130:12)
    at startNext (/usr/local/lib/node_modules/broken-link-checker/node_modules/limited-request-queue/lib/index.js:265:4)
    at RequestQueue.enqueue (/usr/local/lib/node_modules/broken-link-checker/node_modules/limited-request-queue/lib/index.js:64:3)
    at HtmlUrlChecker.enqueue (/usr/local/lib/node_modules/broken-link-checker/lib/public/HtmlUrlChecker.js:103:27)
    at run (/usr/local/lib/node_modules/broken-link-checker/lib/cli.js:489:11)
    at cli.input (/usr/local/lib/node_modules/broken-link-checker/lib/cli.js:147:3)
    at Object.<anonymous> (/usr/local/lib/node_modules/broken-link-checker/bin/blc:3:31)
    at Module._compile (module.js:641:30)
    at Object.Module._extensions..js (module.js:652:10)
    at Module.load (module.js:560:32)
/ #
```

**Comment by stevenvachon** [comment]
Try using the v0.8 branch for now.

**Comment by asaladino** [comment]
Version 0.8 is working. Just in case some is using this as a dependency, you can added it with
`
"dependencies": {
  "broken-link-checker": "stevenvachon/broken-link-checker#v0.8.0",
}
`
Also, nice library 👍. Very helpful.

**Comment by stevenvachon** [comment]
Node v9 is not stable, so I probably won't add support for it in v0.7.x

**Comment by stevenvachon** [comment]
Fixed in v0.7.8
